packages/trader/src/sass/app/modules/mt5/mt5-dashboard.scss

```diff
@@ -396,15 +398,22 @@
 .mt5-real-accounts-display,
 .mt5-demo-accounts-display {
     display: flex;
-    justify-content: center;
+    justify-content: space-between;
     margin-top: 2.4em;
     height: 100%;
 
     @include mobile {
+        overflow-x: hidden;
```

Comment:
why do we need this? 
there shouldn't be any horizontal scrollbar on this element.

Reply:
We need this because a part of the `demo banner` is positioning out of the card and this makes the `dc-tabs` component's width to be wider than its parent and tab titles to be uncentered.